contracts/ICSCore.sol

```diff
@@ -88,11 +96,20 @@ contract ICSCore is IICS02Client, IICS04Channel, IICS02ClientErrors, Initializab
 
         emit ICS04ChannelAdded(clientId, channel);
 
+        bytes32 role = _getLightClientRole(clientId);
```

Comment:
maybe this should be renamed (including the function) to something like light client admin/owner role or something. Role as a name alone is a bit ambiguous.

Reply:
How about `_getLightClientMigratorRole`. Roles can (and should) be very specific rather than being an "owner/admin"

Reply:
Yes, that seems reasonable to me!